plugins/acr/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@janus-idp/backstage-plugin-acr",
-  "version": "1.4.11",
+  "version": "1.7.11",
```

the version in the 1.3 branch is ALSO at 1.7.11 so this will collide. https://github.com/janus-idp/backstage-plugins/blob/release-1.3/plugins/acr/package.json#L3

however 1.7.z is the correct version for the 1.3 branch, as main has moved ACR to https://github.com/backstage/community-plugins/commits/main/workspaces/acr/plugins/acr/package.json where it's at version 1.8.z

Should this be at 1.6.z in 1.2? 
```suggestion
  "version": "1.6.4",
```

---

I missed the version in other branches, your right that makes sense based on the current versioning across the branches. That is:

- main branch 1.8.z (already moved to community-plugins repo)
- release-1.3 branch 1.7.z (currently at 1.7.11, which is correct for this branch)
- 1.2.x branch Since 1.7.11 is used in 1.3, and 1.8.z is in the main branch, the 1.2.z branch should have a version that's lower than 1.7.z to avoid conflicts (Logically speaking, your right). Hence 1.6.4 in 1.2.z